website/blog/2024/11-27-scrape-google-search/index.md

```diff
@@ -0,0 +1,273 @@
+---
+slug: scrape-google-search
+title: 'How to scrape Google search results with Python'
+tags: [community]
+description: 'Learn how to scrape google search results using Crawlee for Python'
+image: ./img/google-search.webp
+authors: [MaxB]
+---
+
+Scraping `Google Search` delivers essential `SERP analysis`, SEO optimization, and data collection capabilities. Modern scraping tools make this process faster and more reliable.
+
+:::note
+One of our community members wrote this blog as a contribution to the Crawlee Blog. If you would like to contribute blogs like these to Crawlee Blog, please reach out to us on our [discord channel](https://apify.com/discord).
+:::
```

Comment:
blank lines are needed for formatters, we dont use them on markdown here just yet, but we will adopt them at some point

```suggestion
:::note

One of our community members wrote this blog as a contribution to the Crawlee Blog. If you would like to contribute blogs like these to Crawlee Blog, please reach out to us on our [discord channel](https://apify.com/discord).

:::
```